src/core-services/product/schemas/schema.graphql

```diff
@@ -24,6 +24,9 @@ input ProductTagsOperationInput {
   "An array of product productIds to which an array of tags will be added"
   productIds: [ID]
 
+  "The shop id"
+  shopId: ID
```

Comment:
Add `!`. It's required because it's in the permission check and the mongo query.